repair/repair.cc

```diff
@@ -256,15 +256,15 @@ tracker::~tracker() {
     _the_tracker = nullptr;
 }
 
-void tracker::start(int id) {
-    _status[id] = repair_status::RUNNING;
+void tracker::start(repair_uniq_id id) {
+    _status[id.id] = repair_status::RUNNING;
 }
 
-void tracker::done(int id, bool succeeded) {
+void tracker::done(repair_uniq_id id, bool succeeded) {
     if (succeeded) {
-        _status.erase(id);
+        _status.erase(id.id);
     } else {
-        _status[id] = repair_status::FAILED;
+        _status[id.id] = repair_status::FAILED;
```

Comment:
Let's call it id.sequence_number to make it clear.

Reply:
But, being a sequence number is just an implementation detail. It is still an id used to identify a repair job.

Reply:
yes, but the uuid is also an id. So we have id, id.id, and id.uuid. It is very unfriendly to someone who isn't aware of the history and doesn't know which of the three choices to use.

Better add a comment to repair_uniq_id too.

Reply:
I added comments to repair_uniq_id.